frontend/src/routing/ListItemLink.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { NavLink, LinkProps } from 'react-router-dom';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+
+interface ListItemLinkProps {
```

Comment:
no comment

Reply:
This file was not created, nor changed by this PR